cla_common/services/__init__.py

```diff
@@ -0,0 +1,35 @@
+class TranslationAdapter(object):
+    _translation_adapter_factory = None
+
+    @classmethod
+    def set_translation_adapter_factory(cls, translation_adapter_factory):
+        cls._translation_adapter_factory = translation_adapter_factory
+
+    @classmethod
+    def get_translation_adapter(cls):
+        if cls._translation_adapter_factory:
+            factory = cls._translation_adapter_factory
+            return factory()
```

Comment:
Are there pros or cons to storing the instantiated adapter? It feels like an efficient thing to do unless there's drawbacks I've not thought of. So that future calls to `TranslationAdapter.get_translation_adapter()` can short-circuit and not have to create and use the factory each time

Reply:
Yeah you are right, I was being lazy because I know that the current factory method returns an already instantiated object i.e return _

I'll update get_translation_adapter to store the return value and use that in future calls.

Good catch

Reply:
This has been update now. Can you review this for me again please @exonian